lib/task-data/schemas/wait-sel-events.json

```diff
@@ -0,0 +1,37 @@
+{
+    "copyright": "Copyright 2017, Dell EMC, Inc.",
+    "definitions": {
+        "alertFilter": {
+            "type": "object",
+            "properties": {
+                "count": {
```

Comment:
I prefer @nortonluo 's idea to set a timeout for this task instead.
I can't enumerate what kind of alert it is required, it might be platform/generation dependent...